src/runtime/basic-runtime.ts

```diff
@@ -243,6 +244,7 @@ export class BasicRuntime implements BasicSubscriptions {
       autoPromptType,
       alwaysShow,
       isClosable,
+      contentType: this.getContentType_(isAccessibleForFree ?? isOpenAccess),
```

Comment:
Want to call this out as a point we should reconfirm with Product — specifically, what defines ContentType?

Previously, AutoPromptManager synonymously used "openaccess content" to mean "content with dismissible prompts", and computed dismissibility based on: isAccessibleForfree (if true), else fallback to `openaccess` productId (if true), else fallback to contribution/subscription model. 

In the above implementation, I have removed the last check for revenue model, and just computed contentType on isAccessibleForFree (if true), else fallback to `openaccess` productId.

Reply:
Spoke offline, the fallbacks we had were specific eng decisions we put in place. Moving forward, `contentType` will solely be calculated from `isAccessibleForFree` and `productId`.